cmd/kubeadm/app/preflight/checks.go

```diff
@@ -593,7 +593,6 @@ func (kubever KubernetesVersionCheck) Check() (warnings, errorList []error) {
 	if k8sVersion.AtLeast(firstUnsupportedVersion) {
 		return []error{errors.Errorf("Kubernetes version is greater than kubeadm version. Please consider to upgrade kubeadm. Kubernetes version: %s. Kubeadm version: %d.%d.x", k8sVersion, kadmVersion.Components()[0], kadmVersion.Components()[1])}, nil
 	}
-
```

Comment:
removing this whitespace is not really needed.
we should try to keep the diff as clean as possible...most of the time.

Reply:
yop